src/Microsoft.AspNetCore.Razor.Language/CodeGeneration/CSharpCodeWritingScope.cs

```diff
@@ -2,22 +2,21 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
-using System.Linq;
 
-namespace Microsoft.AspNetCore.Razor.Language.Legacy
+namespace Microsoft.AspNetCore.Razor.Language.CodeGeneration
 {
     public struct CSharpCodeWritingScope : IDisposable
```

Comment:
So the only reason why this is public today is to prevent boxing. Thinking we should nest this publicly in the `CSharpCodeWriter` since that's the only thing that uses it.